ForgeUnity/Assets/BeardedManStudios/Scripts/Multiplayer Menu/JoinMenu.cs

```diff
@@ -37,8 +38,8 @@ private void Awake()
 			// Init the MainThreadManager
 			MainThreadManager.Create();
 
-			mpMenu = this.GetComponentInParent<MultiplayerMenu>();
-			Settings = mpMenu.Settings;
+			mpMenu = FindObjectOfType<MultiplayerMenu>();
```

Comment:
What's the reason for this change? Ideally the Join menu should be a child of the main multiplayer menu.

Reply:
agree. I moved it in my project which is why I have this... I'll remove this.

Reply:
👍